src/test/java/com/example/parkingapp/service/rate/VipParkingRateWithCustomCurrencyTest.java

```diff
@@ -0,0 +1,50 @@
+package com.example.parkingapp.service.rate;
+
+import com.example.parkingapp.config.AcceptedCurrencies;
+import com.example.parkingapp.domain.CurrencyConversionRate;
+import org.assertj.core.api.WithAssertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.Collection;
+
+@RunWith(Parameterized.class)
+public class VipParkingRateWithCustomCurrencyTest implements WithAssertions {
+    private static CurrencyConversionRate defaultCurrency;
+
+    static {
+        defaultCurrency = new CurrencyConversionRate();
+        defaultCurrency.setConversionRate(new BigDecimal("0.75"));
```

Comment:
conversion rate względem czego?

Reply:
Ten trochę naiwny i prosty mechanizm działa tak, że istnieje obsługiwana waluta z konwersją 1.0 (domyślnie to złotówka) i kolejne waluty w aplikacji dodaje się uwzględniając kurs w stosunku do niej, np. Gdybyśmy chcieli dodać dolara amerykańskiego to ustawiamy konwersję na 0.29438. Nie jest to zbyt elastyczne (zakłada że ceny parkowania będą identyczne we wszystkich walutach), ale umożliwia prostą obsługę nowych walut.